src/dir.targets

```diff
@@ -174,6 +188,33 @@
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="One or more tests failed while running tests from '$(MSBuildProjectName)' please check log for details!" />
   </Target>
 
+  <Target Name="CheckTestCategories"
+          AfterTargets="Test"
```

Comment:
Please remove AfterTargets="Test" and simply make this target a dependency of only RunTestsForProject.